template-vue-ts/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": true,
+    "jsx": "preserve",
+    "sourceMap": true,
+    "lib": ["esnext", "dom"],
+    "plugins": [{ "name": "@vuedx/typescript-plugin-vue" }]
```

```suggestion
    "plugins": [{ "name": "@vuedx/typescript-plugin-vue" }]
```
It's not yet stable I think, let's don't include it for now.